pkg/iac/terraform/state/backend/tfcloud_reader.go

```diff
@@ -36,7 +35,12 @@ type TFCloudBackend struct {
 }
 
 func NewTFCloudReader(client pkghttp.HTTPClient, workspaceId string, opts *Options) (*TFCloudBackend, error) {
-	req, err := http.NewRequest(http.MethodGet, fmt.Sprintf("%s/workspaces/%s/current-state-version", TFCloudAPI, workspaceId), nil)
+	// Assume if it was not set that the end-point is Terraform Cloud
+	if opts.TFCloudAPI == "" {
+		opts.TFCloudAPI = "https://app.terraform.io/api/v2"
+	}
```

Comment:
This default should be defined in the command args definition

```
fl.StringVar(&opts.BackendOptions.TFCloudAPI,
		"tfc-api",
		"https://app.terraform.io/api/v2",
```

Reply:
Ok.  Thanks for the pointer on this.  I prefer this way, too.